docs/Hosts/Spawn-Hosts.md

```diff
@@ -55,6 +55,149 @@ lifetime up to 30 days past host creation.
+
+For unexpirable hosts, Evergreen has introduced a new feature called a host sleep schedule, which allows you to control
+when you'd like your unexpirable host to be automatically turned on or off. A sleep schedule lets you choose recurring
+times of the week when you want your host to be on, and when it's okay for it to be turned off. For example, you could
+set a sleep schedule where your host is on during your work hours from 10 am to 6 pm between Monday and Friday and
+otherwise let it turn it off overnight and during weekends. Setting a sleep schedule ensures your host is on when you're
+working, while also stopping the host when it's not being actively used. This is important to ensure that hosts are
+being efficiently utilized and to avoid over-spending on long-lived AWS resources when they're idle.
+
+Please note that even though Evergreen will automatically stop and start your host at the configured times, you can
+still manually stop and start your host whenever you need. It's recommended that if you do this, you also [set a
+temporary exemption](#temporary-exemptions) to ensure that you can use your host without being interrupted when the
+sleep schedule tries to automatically stop/start your host.
+
+In the Spruce UI, you can pick a sleep schedule for your host when creating the unexpirable host or editing an existing
+unexpirable host:
+
+![Sleep schedule](../images/set_sleep_schedule.png)
+
+The sleep schedule above will automatically turn the host on from Monday to Friday.
+
+From the create/edit host modal, you can pick which days you'd like the host to be on, and the time that you want your
+host to be on for those days. The time zone for the host sleep schedule will be based on the time zone you choose in
+your [Spruce preferences](https://spruce.mongodb.com/preferences/profile).
+
+#### ***Important Note***: This Feature is in Beta Testing!
+
+<!-- TODO (DEVPROD-4055): do not merge until comms are ready and a timeline for beta/full rollout is chosen.  -->
+
+This feature is being rolled out in phases and is currently in beta testing. While it's in beta testing, _using the
+sleep schedule feature is an opt-in_. During this beta testing period, you have until `<DATE>` to set a sleep schedule
+for any of your existing unexpirable hosts. Until then, you should set a sleep schedule for any of your existing
+unexpirable hosts. On top of setting a sleep schedule, you also have the option to participate in the beta test. If you
+opt into the beta test, the sleep schedule that you set will take effect on your host, stopping and starting your host
+according to the schedule you configure; if you do not opt in, your sleep schedule will be set but will have no effect
+on your host. You can opt in or out freely during the beta test. **It's highly recommend for you to set a sleep schedule
+for your unexpirable host(s) and opt into the beta to verify that your sleep schedule is configured the way you want.**
```

Comment:
I think this would benefit by being broken up into two bullet points. Something along the lines of: 

During the beta testing period you can do one of two things: 

1. Select a schedule to that will kick in after the beta testing period.... etc... 

2. Opt in to the beta test.... etc...

3. Request a permanent exception...

Reply:
Added some bullet points.